raycicmd/rayci_pipeline.go

```diff
@@ -33,6 +33,12 @@ var (
 	}
 	blockStepDropKeys = []string{"tags"}
 
+	triggerStepAllowedKeys = []string{
+		"trigger", "label", "async", "build", "depends_on",
```

Comment:
will this also include they keys under `build` like `commit`, `branch`, or `env`?

Reply:
it will yes